.github/workflows/ci.yml

```diff
@@ -7,6 +7,26 @@ on:
     branches: [ main ]
 
 jobs:
+  arm_test:
+    name: CMake test on ARM (without otlp-exporter)
```

Comment:
should we do this with otlp-exporter?

Reply:
was just testing the runner.
testing otlp now.